src/components/OSCALInventoryItem.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { styled } from "@mui/material/styles";
+import Grid from "@mui/material/Grid";
+import Typography from "@mui/material/Typography";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell, { tableCellClasses } from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import OSCALResponsibleRoles from "./OSCALResponsibleRoles";
+import OSCALImplementedComponents from "./OSCALImplementedComponents";
+import StyledTooltip from "./OSCALStyledTooltip";
+
+const StyledTableCell = styled(TableCell)(({ theme }) => ({
+  [`&.${tableCellClasses.head}`]: {
+    backgroundColor: theme.palette.primary.main,
+    color: theme.palette.common.white,
+  },
+  textAlign: "left",
+  minWidth: "10em",
+}));
+
+const StyledTableRow = styled(TableRow)(({ theme }) => ({
+  // Use hover color for even numbered rows
+  "&:nth-of-type(even)": {
+    backgroundColor: theme.palette.action.hover,
+  },
+}));
+
+export default function OSCALInventoryItem(props) {
+  if (!props.inventoryItem) {
+    return null;
+  }
+
+  return (
+    <>
+      <Grid item xs={12}>
+        <Typography variant="subtitle1" gutterBottom component="div">
+          {props.inventoryItem?.description}
+        </Typography>
+        <Typography variant="body2">{props.inventoryItem?.remarks}</Typography>
+      </Grid>
+      <Grid item xs={12}>
+        <TableContainer sx={{ maxHeight: "15em" }}>
+          <Table aria-label="Inventory Items" sx={{ height: "max-content" }}>
+            <TableHead>
+              <TableRow>
+                <StyledTableCell>Item</StyledTableCell>
+                <StyledTableCell>Value</StyledTableCell>
+                <StyledTableCell>Responsible Parties</StyledTableCell>
+                <StyledTableCell>Implemented Components</StyledTableCell>
+                <StyledTableCell>Remarks</StyledTableCell>
```

Comment:
So `inventory-items` are organized into separate lists. I was creating separate tables to organize each of these lists. This `TableHead` sits at the top of these lists, not each individual item. To avoid the user from having to scroll up the page to see what the table head is, I included it for each list. I wouldn't mind making this change, if this makes more sense. Although, there can be 1-infinity `inventory-items`, so in the case there a ton, the user would have to do a lot of scrolling.

Reply:
Looking at `Components` we have usually just head a `header` then a new row for each `item`. I do not think we need a new header for each inventory item.